ansys_optical_automation/application/radiance_from_view.py

```diff
@@ -0,0 +1,32 @@
+from ansys_optical_automation.speos_process.speos_sensors import RadianceSensor
+
+
+def main():
+    Command.Execute("FlyThroughCamera")
+    InputHelper.PauseAndGetInput("Adjust the window to the desired point of view and validate")
+    Camera = GetActiveWindow().GetCameraFrame()
+    HFOV = round(GetActiveWindow().HorizontalFieldOfView * 180 / 3.14, 0)
+    VFOV = round(GetActiveWindow().VerticalFieldOfView * 180 / 3.14, 0)
+    DatumCS = DatumOriginCreator.Create(Camera)
+    cs = DatumCS.CreatedOrigin
+    cs.SetName("ObserverOriginRadianceSensor")
+
+    Radiance = RadianceSensor("Radiance_from_View", SpeosSim, SpaceClaim)
+    Radiance.set_observer_type("observer")
+    Radiance.set_type("colorimetric")
```

Comment:
in-correct result running the script, "colorimetric" has not been used to define sensor